src/Halogen/Aff/Driver/Eval.purs

```diff
@@ -72,14 +69,14 @@ evalM
    . Renderer r
   -> Ref (DriverState r s f act ps i o)
   -> HalogenM s act ps o Aff
-  ~> Aff
+       ~> Aff
```

Comment:
These aren't ideal; we could rewrite them without the natural transformation operator, or live with the indentation. Thoughts @natefaubion @garyb?

Reply:
I would think this is perhaps not entirely intentional? I suppose outside of arrow-like operators it is odd to split them across lines though. Any thoughts Nate?

I would definitely rewrite rather than use indentation here if we have no other choice.

Reply:
This is expected. `~>` is an operator and doesn't have the same precedence as quantifiers. This goes double if you ever try to use `~>` in any position besides the final return value.

Reply:
It might be "expected" but it's nasty! 😄 By "not entirely intentional" here I meant more along the lines of, it might be following the rules, but the consequence is something unwanted.

Reply:
If you can think of a rule that doesn't hardcode an unqualified `~>`, then I'm all ears ;P

Reply:
I guess a question I have is why does it indent based on precedence? Or is the precedence we're talking about here syntax vs identifiers?

Reply:
In this case, yes, it is syntax vs identifiers. Type expressions will be indented past the arrow. We do indent based on operator precedence though elsewhere!